packages/suite/src/reducers/suite/protocolReducer.ts

```diff
@@ -27,6 +50,16 @@ const protocolReducer = (state: State = initialState, action: Action): State =>
                 draft.sendForm.amount = action.payload.amount;
                 draft.sendForm.shouldFillSendForm = false;
```

Comment:
would be nice to unify naming 😇  and then types (`shouldFill` logic)

Reply:
https://github.com/trezor/trezor-suite/pull/4572/commits/909b8f305a88561f85f279f751fb2fecd21d1f7d